content/en/docs/install-operate/third-party/install-tools.md

```diff
@@ -36,15 +36,6 @@ information about how to download and install Helm.
 
 <a class="btn btn-primary" href="https://helm.sh/docs/intro/install/" role="button" aria-label="View Installing Helm Guide">View Installing Helm Guide</a>
 
-
-## Puppet
-
-A [Puppet](https://puppet.com/) module for Falco, `sysdig-falco`, is available on [Puppet Forge](https://forge.puppet.com/sysdig/falco/readme).
-
-## Ansible
-
-[@juju4](https://github.com/juju4/) has helpfully written an [Ansible](https://ansible.com) role for Falco, `juju4.falco`. It's available on [GitHub](https://github.com/juju4/ansible-falco/) and [Ansible Galaxy](https://galaxy.ansible.com/juju4/falco/). The latest version of Ansible Galaxy (v0.7) doesn't work with Falco 0.9, but the version on GitHub does.
-
 ## kubectl
```

Comment:
Kubectl is not a tool to manage Falco, but Pods in general. IMHO, it's an implicit tool of any K8s cluster and doesn't add value here.